.gitignore

```diff
@@ -1,4 +1,2 @@
 build/
 .vscode
-models/*.bin
```

Comment:
why remove models/*.bin ?

Reply:
don't remember the exact reason. probably it was a temporary change then I forgot about re-adding it. It should definitely be added again.